Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -333,3 +434,35 @@ private fun darken(@ColorInt normalColor: Int, factor: Float = .2f): Int {
   hsv[2] *= 1f - factor // value component
   return Color.HSVToColor(hsv)
 }
+
+private fun convertToBitmap(drawable: Drawable): Bitmap? {
```

Comment:
this can be in some util file, so other components can use it if required.

Reply:
I'd keep here for now and move only when we actually need it for other components. Rule of three :)